platform/android/llapp_main.lua

```diff
@@ -68,7 +68,6 @@ end
 --- @todo Take care of this on extraction instead.
 -- Cf. <https://github.com/koreader/koreader/issues/5347#issuecomment-529476693>.
 android.execute("chmod", "755", "./sdcv")
-android.execute("chmod", "755", "./tar")
```

Comment:
Should we stop building tar on Android then?

Reply:
Please